projects/kserve/kserve-incubation-dd.md

```diff
@@ -320,23 +308,50 @@ N/A
 - [ ] **Clearly defined and discoverable process to report security issues.**
 
 <!-- (TOC Evaluation goes here) --> 
+KServe’s security vulnerability reporting process is outlined at [kserve/SECURITY.md at master · kserve/kserve](https://github.com/kserve/kserve/blob/master/SECURITY.md). The mail id is currently under lfaidata foundation, it is recommended to move to a CNCF domain at the appropriate time.
+
+**TODO for KServe Team:**
+1. Add SECURITY.md to all relevant repos and not just kserve/kserve.
+2. Link SECURITY.md from all contributor guides across github and the KServe website. For eg: https://github.com/kserve/community?tab=readme-ov-file#questions-and-issues, https://kserve.github.io/website/0.15/developer/developer/ and https://github.com/kserve/community/blob/main/CONTRIBUTING.md#issues. All of them does not seem to have uniform information and how to report security vulnerabilty should be part of them.
 
 - [ ] **Enforcing Access Control Rules to secure the code base against attacks (Example: two factor authentication enforcement, and/or use of ACL tools.)**
 
 <!-- (TOC Evaluation goes here) --> 
+Access control is enforced via OWNERS file [kserve/OWNERS at master · kserve/kserve](https://github.com/kserve/kserve/blob/master/OWNERS) for Kserve repo and most of the other repos. (would have been better to name this CODEOWNERS), and the respective roles are specified in https://github.com/kserve/community/blob/main/membership.md#community-membership.
+
+Members of KServe organization are required to enable two factor authentication [community/membership.md at main · kserve/community](https://github.com/kserve/community/blob/main/membership.md#requirements) in their Github account.
+
+**TODO for KServe Team:**
+1. Add OWNERS file to kserve/community repo.
```

Comment:
It's located here: https://github.com/kserve/community/blob/main/.github/CODEOWNERS